### PR TITLE
Add lighttpd

### PR DESCRIPTION
**Pull request by sdvillal**
This is a refurbished recipe from the one at defaults. We are already using this package, let's see if it also builds in mac...

### CODE REVIEW

File: recipes/lighttpd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  files:
+    - lighttpd.conf
+  commands:
+    - lighttpd -h
+    - lighttpd -v
+    - lighttpd -t -f lighttpd.conf
+
+about:
+  home: http://www.lighttpd.net/
+  license: BSD
```
Comment (jakirkham) [inline recipes/lighttpd/meta.yaml]: Please add version of BSD (e.g. `BSD 3-Clause` for the 3-Clause license).

File: recipes/lighttpd/build.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+./configure \
+    --with-sysroot=$PREFIX \
+    --prefix=$PREFIX
+make
+make install
```
Comment (jakirkham) [inline recipes/lighttpd/build.sh]: Is there a `make check` we can run?

**Comment by scopatz** [comment]
pinging this @sdvillal

**Comment by sdvillal** [comment]
@jakirkham Thanks for the review. I have enabled tests and added a few features to the compiled package. Unfortunately, I could not get the green tick for mac builds - a lot of core tests failing. So for the time being I have disabled mac builds.

**Pull request merged**